mw-webapp/src/logic/allUsersPage/AllUsersPage.tsx

```diff
@@ -187,40 +184,22 @@ export const AllUsersPage = observer(() => {
             onChange={(value) => setMentorStatus(value)}
           />
         </HorizontalContainer>
-        <HorizontalContainer>
-          <Tooltip
-            position={PositionTooltip.LEFT}
-            content={LanguageService.allUsers.filterBlock.cardViewTooltip[language]}
-          >
-            <button
-              className={styles.iconView}
-              onClick={() =>
-                updateAllUsersPageSettings({view: View.Card})}
-            >
-              <Icon
-                size={IconSize.MEDIUM}
-                name={"GridViewIcon"}
-                className={clsx(allUsersPageSettings.view === View.Card && styles.activeView)}
-              />
-            </button>
-          </Tooltip>
-          <Tooltip
-            position={PositionTooltip.LEFT}
-            content={LanguageService.allUsers.filterBlock.tableViewTooltip[language]}
-          >
-            <button
-              className={styles.iconView}
-              onClick={() =>
-                updateAllUsersPageSettings({view: View.Table})}
-            >
-              <Icon
-                size={IconSize.MEDIUM}
-                name={"TableViewIcon"}
-                className={clsx(allUsersPageSettings.view === View.Table && styles.activeView)}
-              />
-            </button>
-          </Tooltip>
-        </HorizontalContainer>
+        <ViewSwitcher
+          view={allUsersPageSettings.view}
+          setView={(view) => updateAllUsersPageSettings({view})}
+          options={[
+            {
+              view: View.Card,
+              tooltipContent: LanguageService.allUsers.filterBlock.cardViewTooltip[language],
+              iconName: "GridViewIcon",
+            },
+            {
+              view: View.Table,
+              tooltipContent: LanguageService.allUsers.filterBlock.tableViewTooltip[language],
+              iconName: "TableViewIcon",
+            },
+          ]}
+        />
```

Comment:
What do you think about move it to const and reuse it in all the files?

Reply:
Content variable is different on each page

Reply:
Well we can move OptionItems and reuse them/
For example optinCard, optionTable and etc

Reply:
Done